### ISSUE TITLE
Support Private Browsing mode

### ISSUE THREAD

**Issue by marniepw** [mozilla/notes#665]
[Description]: "Notes" is a good idea, but not working in private browsing mode. Therefore not usable for me.

[Affected versions]: Unknown; from January survey feedback.

[Prerequisites]:

[Steps to reproduce]:

[Expected result]:

[Actual result]:

[Additional Notes]:

**Comment by Natim** [comment]
We cannot reproduce this, since it seems to be working here. (Toolbar button and shortcut.)

**Comment by Softvision-MariusComan** [comment]
I think that the users are encountering the following issue here: https://bugzilla.mozilla.org/show_bug.cgi?id=1430070. Because on a private window the sidebar is closed by default and the address bar is focused.

**Comment by Natim** [comment]
I just tried on Linux and Nightly and I can use the shortcut even while the address bar is focused.

**Comment by Softvision-MariusComan** [comment]
I have retested this issue and it seems that is no longer reproducible on the latest Nightly build (20180207220120 - Build ID). However I've still managed to reproduce it on Release and Beta channels.
Considering this, using the Mozregression tool, I have managed to find the fix:

First good revision: 48bca44fe6a6e88f941a1a5ab2a41a041346fa21
Last bad revision: df6d2c3ee671e81f1a8b4f6d13197eee52db9680
Pushlog: https://goo.gl/dvs3Vx

It looks like the [Bug 1435530](https://bugzilla.mozilla.org/show_bug.cgi?id=1435530) has fixed this issue.
